MapleServer2/Servers/Game/DungeonManager.cs

```diff
@@ -0,0 +1,113 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using MapleServer2.Types;
+
+namespace MapleServer2.Servers.Game
+{
+    public class DungeonManager
+    {
+        public readonly Dictionary<int, DungeonSession> DungeonSessionList;
+        private int SessionId = 0;
+        private List<int> RecyclableSessionIds;
+        private List<int> RecyclableMapInstanceIds;
```

Comment:
```suggestion
        private readonly List<int> RecyclableSessionIds;
        private readonly List<int> RecyclableMapInstanceIds;
```

Reply:
I was wondering, I .add to the list outside of the constructor, why can it be readonly then? :O

Reply:
You can manipulate the list but you can't set to another value!

Reply:
```
RecyclableSessionIds.Add(currentDungeonSessionId); // Ok!

RecyclableSessionIds = new List<int>(); // Not ok!
```

Reply:
ooh, interesting, I thought it was like const.